### ISSUE TITLE
External services in appmap are missing events

### ISSUE THREAD

**Issue by ahtrotta** [getappmap/appmap-js#956]

**Comment by brikelly** [comment]
@ahtrotta is this still in progress?

**Comment by ahtrotta** [comment]
Yeah, this is causing some of the bugs Kevin pointed out in his sequence diagram doc. And it's what I talked with Dustin about earlier. I should have a fix for it by the end of the day today.

**Comment by ahtrotta** [comment]
Fixed [here](https://github.com/getappmap/appmap-js/pull/982/commits/f1a05c767c9d129e2231432ac52efffbf11a1c03)
